### PR TITLE
Migrated rock list to recycler view

### LINKED ISSUE #305: Migrate rock list to RecyclerView
Subtask of #291

### PR DESCRIPTION
**Pull request by paetz**
Resolves #305

### CODE REVIEW

File: app/src/main/java/com/yacgroup/yacguide/database/Rock.kt [hunk 1]
```diff
@@ -21,23 +21,20 @@ import androidx.room.Entity
...
 
 @Entity
-class Rock {
-
-    @PrimaryKey
-    var id: Int = 0
-
-    var nr: Float = 0.toFloat()
-    var type: Char = ' '   // 'G' = Rock, 'M' = Massif, 'H' = Cage, 'B' = Boulder, 'N' = unofficial
-    var status: Char = ' ' // 'X' = prohibited, 'Z' = temporarily prohibited, 'T' = partly prohibited
```
Comment (chrgernoe) [inline app/src/main/java/com/yacgroup/yacguide/database/Rock.kt:31]: Why did you remove these comment?

**Review by chrgernoe — APPROVED** [review]

File: app/src/main/java/com/yacgroup/yacguide/database/Rock.kt (hunk 1 above)
Comment (paetz) [reply app/src/main/java/com/yacgroup/yacguide/database/Rock.kt:31]: It's obvious since the constants are defined below.
